pulp_ansible/app/galaxy/views.py

```diff
@@ -189,6 +190,7 @@ def post(self, request, path):
         return OperationPostponedResponse(async_result, request)
 
 
+@extend_schema_view(get=extend_schema(operation_id="api_v2_collection_versions_list"))
```

Comment:
I'm not too good glad that the liveapi is part in our open API, but I guess it's too late anyway.